.github/workflows/super-linter.yml

```diff
@@ -27,7 +27,6 @@ jobs:
           VALIDATE_ALL_CODEBASE: false 
           DEFAULT_BRANCH: main
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          VALIDATE_PHP_BUILTIN:  true
```

Comment:
I was experimenting with having both PHPCS and PHP base in there. It seems overkill to have both at this point, especially since the base PHP is "linting" in as much as it validates runnable code, but doesn't do any real linting. It should probably go in its own commit/branch.